### PR TITLE
Update save page tracks to use new api client

### PR DESCRIPTION
**Pull request by jowlee**
### Trello Card Link
https://trello.com/c/A7BOsv5J/1476-port-the-new-api-into-the-dapp

### Description
Moves the tracks from from the favorites page to be sourced from the new api

### Dragons

Is there anything the reviewer should be on the lookout for? Are there any dangerous changes?
na

### How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide repro instructions & any configuration.
Ran locally against a local DP w/ DB pointing to a prod snapshot

### CODE REVIEW

File: src/services/audius-api-client/AudiusAPIClient.ts [hunk 1]
```diff
@@ -18,7 +18,8 @@ const ENDPOINT_MAP = {
   playlistFavoriteUsers: (playlistId: string) =>
     `/playlists/${playlistId}/favorites`,
   userByHandle: (handle: string) => `/users/handle/${handle}`,
-  userTracksByHandle: (handle: string) => `/users/handle/${handle}/tracks`
+  userTracksByHandle: (handle: string) => `/users/handle/${handle}/tracks`,
+  favoritedTracks: (userId: string) => `/users/${userId}/favorites/tracks`
```
Comment (raymondjacobson) [inline src/services/audius-api-client/AudiusAPIClient.ts:22]: nit, let's call userFavoritedTracks

File: src/services/audius-api-client/AudiusAPIClient.ts [hunk 2]
```diff
@@ -349,6 +357,40 @@ class AudiusAPIClient {
...
+      params
+    )
+    const favoritedTrackResponse: APIResponse<
+      { timestamp: string; item_type: 'track'; item: APITrack }[]
+    > = await this._getResponse(endpoint)
+    const adapted = favoritedTrackResponse.data
+      .map(({ item, ...props }) => ({
+        timestamp: props.timestamp,
+        track: adapter.makeTrack(item)
+      }))
```
Comment (raymondjacobson) [inline src/services/audius-api-client/AudiusAPIClient.ts:389]: perhaps we should pull the adapter out into a makeFavoritedTracks. wdyt?

**Review by raymondjacobson — APPROVED** [review]

File: src/services/audius-api-client/AudiusAPIClient.ts (hunk 2 above)
Comment (raymondjacobson) [reply src/services/audius-api-client/AudiusAPIClient.ts:389]: actually, feel free to just merge. ill do this in my PR because im adding an activity type

**Comment by audius-infra** [comment]
Preview this change https://frontend-demo.audius.co/jowlee-track-favorites

**Pull request merged**